src/simplewallet/simplewallet.cpp

```diff
@@ -501,9 +501,9 @@ bool simple_wallet::set_default_priority(const std::vector<std::string> &args/*
     else
     {
       priority = boost::lexical_cast<int>(args[1]);
-      if (priority != 1 && priority != 2 && priority != 3)
+      if (priority < 1 || priority > 4)
```

Comment:
Shouldn't this be `priority < 0 || priority > 4` ? According to the message priority 0 seems to be valid.